static/css/frame.scss

```diff
@@ -605,3 +604,147 @@ body {
   background-repeat: no-repeat;
   background-position: center;
 }
+
+// CSS used for Annotation Text Tool
+.text-select {
+  -moz-appearance: none; //sass-lint:disable-line no-vendor-prefixes
+  appearance: none;
+  background-color: $light-default;
+  background-image: url("../img/icon-dropdown.svg");
+  background-position: right 8px top 8px;
+  background-repeat: no-repeat;
+  background-size: 10px auto;
+  border-radius: $border-radius;
+  border: 1px solid $light-border;
+  color: $dark-default;
+  cursor: pointer;
+  font-size: 14px;
+  font-weight: 400;
+  height: 26px;
+  margin: 12px 5px 12px 15px;
+  outline: none;
+  padding: 0 24px 0 4px;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  transition: background 150ms $bezier, border 150ms $bezier;
+  user-select: none;
+  white-space: nowrap;
+  vertical-align: middle;
+
+  &:hover,
+  &:focus {
+    background-color: $light-hover;
+    border-color: $light-border-active;
+  }
+
+  &:active {
+    background-color: $light-active;
+    border-color: $light-border-active;
+  }
+
+  option {
+    padding: 0 40px 0 4px;
+  }
+}
+
+.text-button {
+  background-image: url("../img/annotation-text.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &:hover {
+    background-color: $light-hover;
+  }
+}
+
+.text-tool-container {
+  z-index: 4;
+}
+
+// Text Sizes used in the text tool
+$large: 36px;
+$medium: 24px;
+$small: 16px;
+
+// Colors used in the text tool
+$red: rgb(231, 76, 60);
+$green: rgb(46, 204, 113);
+$blue: rgb(52, 152, 219);
+$yellow: rgb(255, 255, 0);
+$purple: rgb(142, 68, 173);
+$sea-green: rgb(26, 188, 156);
+$grey: rgb(52, 73, 94);
+
+$bg-black: rgba(0, 0, 0, 0.6);
+$bg-white: rgba(255, 255, 255, 0.7);
+
+@mixin type-box($size) {
+  padding: 4px $size * 0.5;
+  font-size: $size;
+  height: $size + 8;
```

Comment:
Bummer that we need to set this so that the empty div looks ok. (Definitely not an issue if a text input is used instead of a contenteditable div.)  The values here are actually too small; the cursor and text are not middle aligned.  But let's not set it here this way.  Once the input div is rendered, set its height to its line-height plus top and bottom padding.